src/pages/ContributionPage.tsx

```diff
@@ -46,12 +47,25 @@ const ContributionPage = () => {
 									{children}
 								</a>
 							),
+							img: ({ children, ...props }) => {
+								let rawUrl = props.src ?? ''
+
```

Comment:
It should be instead converting only relative urls (i.e. - if  it doesn't start with `https://`  then append these urls  
with  `https://raw.githubusercontent.com/thinkingmachines/unicef-ai4d-research-bank/main/`. 
Example:
`assets/fork1.png` --> `https://raw.githubusercontent.com/thinkingmachines/unicef-ai4d-research-bank/main/assets/fork1.png`

Reply:
Also - you can edit the `catalog-contribution.md ` to remove the hard coded image urls (`https://raw.githubusercontent...`)  there as well. It should render on both the github page as well as the research bank site.